generators/new_collateral/config/parser.go

```diff
@@ -19,22 +20,28 @@ func NewParser() parser {
 	return parser{}
 }
 
+var (
+	ErrorDecodingConfigFile          = errors.New("error decoding config file")
+	ErrorParsingExporterMetadata     = errors.New("error parsing exporter metadata from config file")
+	ErrorParsingTransformerExporters = errors.New("error parsing transformer exporters from config file")
+)
+
 func (parser) ParseCurrentConfig(configFilePath, configFileName string) (types.TransformersConfig, error) {
 	var tomlConfig types.TransformersConfigForToml
 	fullConfigFilePath := helpers.GetFullConfigFilePath(configFilePath, configFileName)
 	_, decodeErr := toml.DecodeFile(fullConfigFilePath, &tomlConfig)
 	if decodeErr != nil {
-		return types.TransformersConfig{}, fmt.Errorf("error decoding config file: %w", decodeErr)
+		return types.TransformersConfig{}, fmt.Errorf("%w: %s", ErrorDecodingConfigFile, decodeErr)
```

Comment:
Shouldn't both of the format strings be %w - since ErrorDecodingConfigFile is an error, but so is decodeErr?